sass/_ResponsiveTabs.scss

```diff
@@ -63,7 +63,8 @@
 
 .coveo-tab-list-container {
   max-width: 80%;
-  z-index: 11;
+  z-index: 999;
```

Comment:
A bit of a smaller value like 20 (same as ResponsiveFacet dropdown) should be sufficient?